packages/wrangler/src/generate.ts

```diff
@@ -0,0 +1,33 @@
+import { DeprecationError } from "./errors";
+import type { Argv, ArgumentsCamelCase } from "yargs";
+
+interface GenerateArgs {
+  name: string;
+  template: string;
+}
+
+export const generateOptions = (yargs: Argv) => {
```

Comment:
could these be regular functions? we try to avoid the assignment style unless it's necessary 
```suggestion
function generateOptions(yargs: Argv) {
```